queue-api/src/main/java/com/example/queueapi/application/global/utils/CookieUtils.java

```diff
@@ -0,0 +1,22 @@
+package com.example.queueapi.application.global.utils;
+
+import jakarta.servlet.http.Cookie;
+import jakarta.servlet.http.HttpServletRequest;
+
+import java.util.Arrays;
+
+public class CookieUtils {
+
+    public static String getCookie(HttpServletRequest request, String target) {
+        Cookie[] cookies = request.getCookies();
+        if (cookies != null) {
+            return Arrays.stream(cookies)
+                    .filter(it -> it.getName().equalsIgnoreCase(target))
+                    .findFirst()
+                    .orElse(new Cookie(target, ""))
```

Comment:
```suggestion
                    .orElse("")
```

쿠키가 없다면 그냥 빈 문자열 `""` 또는  Optional  처리도 가능하지 않을까요?


함수 반환값을 `Optional<String>` 으로 처리하면

호출하는 쪽에서 `isPresent()` 로 체크하는것도 좋을것 같아서요 ㅎㅎ

```java
public static Optional<String> getCookieValue(HttpServletRequest request, String target) {
    if (request.getCookies() == null) return Optional.empty();

    return Arrays.stream(request.getCookies())
            .filter(cookie -> cookie.getName().equals(target))
            .map(Cookie::getValue)
            .findFirst();
}

String value = CookieUtils.getCookieValue(request, "token").orElse("");

```
이런 느낌으로요~

Reply:
넵! 다음 이슈에서 변경하겠습니다!!